modules/nf-core/idemux/meta.yml

```diff
@@ -0,0 +1,50 @@
+name: idemux
+description: Demultiplex paired-end FASTQ files from QuantSeq-Pool
+keywords:
+  - demultiplex
+  - lexogen
+  - fastq
+tools:
+  - idemux:
+      description: |
+        A Lexogen tool for demultiplexing and index error correcting fastq files. Works with Lexogen i7, i5 and i1 barcodes.
+      homepage: https://github.com/Lexogen-Tools/idemux
+      documentation: https://github.com/Lexogen-Tools/idemux
+      tool_dev_url: https://github.com/Lexogen-Tools/idemux
+      licence: ["LEXOGEN"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'NVQ', lane:1 ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files
+      pattern: "Undetermined_S*.fastq.gz"
+  - samplesheet:
+      type: file
+      description: Input samplesheet
+      pattern: "*.{csv}"
+output:
+  - versions:
+      type: file
+      description: File containing software versions
```

Comment:
Can you make the outputs be in the same order as the main.nf (i.e. versions to the bottom)